packages/retail-ui-extensions/src/components/FormattedTextField/FormattedTextField.ts

```diff
@@ -1,7 +1,8 @@
 import {createRemoteComponent} from '@remote-ui/core';
+import {TextFieldProps} from 'components/TextField';
 import type {AutoCapitalizationType} from '../shared';
 
-export type InputType =
+type InputType =
```

Comment:
I think we should keep this exported, partners might want to have access to this type.